### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by amit-gosavi**
no-work-item

### CODE REVIEW

**Review by ashwini4code — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,32 +1,64 @@
...
-       --set secret.applicationToken=$(cat /tmp/aat) --create-namespace --namespace=k8s-info-cron-service 
-       --set cronString='0 */1 * * *' 
+   helm upgrade --install k8s-info-cron-service ./k8s-info-collector-charts/k8s-info-cron-service \
+   --set secret.applicationToken=$AAT_TOKEN --create-namespace --namespace=k8s-info-cron-service \
+   --set cronString='0 */1 * * *'
    ```
-6. Learn more about Parts Discovery by visiting https://devrev.ai/docs/product/parts
+6. Learn more about Parts Discovery [here](https://devrev.ai/docs/product/parts)
+
+## How to Generate Application access token (AAT)?
```
Comment (ashwini4code) [inline README.md:41]: Should we move these instructions to bullet 3 where we say the application will generate the AAT?  We should also probably change it to app will generate PAT?

File: README.md [hunk 2]
```diff
@@ -1,32 +1,62 @@
...
-        helm upgrade --install  k8s-info-cron-service ./k8s-info-collector-charts/k8s-info-cron-service  
-        --set secret.applicationToken=$AAT_TOKEN --create-namespace --namespace=k8s-info-cron-service
+3. Generate Application access token (`AAT`)
+   1. To generate application access token user need to use Personal Access Token(`PAT`) generated through DevRev application.
+      1. How to Generate `PAT`? 
+         1. After logging in to DevRev app click on the organization icon and select `DevOrg Settings`.
+         2. Select `Account` tab. 
+         3. Scroll to the section `Personal Access Token` and generate `New Token` (Please note down the PAT as its visible only one time). 
+         4. For some reason if you lose PAT create new one with above steps
+   2. Use below curl command to generate `AAT`. Make sure you replace `<PAT>` in command with the actual `PAT` obtained in above step. From response copy the `access_token` which is your `AAT`.
```
Comment (ashwini4code) [inline README.md:28]: identation issue?

File: README.md [hunk 3]
```diff
@@ -1,32 +1,62 @@
...
 1. Clone this repository
    ``` 
-       git clone https://github.com/devrev/k8s-info-collector-charts.git 
+   git clone https://github.com/devrev/k8s-info-collector-charts.git 
    ```
 
-2. Log in to the DevRev App using app.devrev.ai and navigate to the connections menu to connect to Kubernetes
-
-3. The App will generate the applocation access token required and surface the below command with the required credentials to execute on your cluster.
+2. Log in to the DevRev [Application](https://app.devrev.ai) and navigate to the `connections` menu to connect to your Kubernetes
```
Comment (ashwini4code) [inline README.md:19]: maybe we remove this for now, since it doesn't exist yet?

Comment (amit-gosavi) [reply README.md:19]: right

**Pull request merged**